spec/views/featured_attachments/_featured_attachment.html.erb_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe "featured_attachments/featured_attachment.html.erb" do
+  describe "official document metadata" do
+    let(:edition) do
+      create(:edition, document_type: build(:document_type, attachments: "featured"))
+    end
+
+    def render_with_attachment(attributes)
+      attachment = create(:file_attachment_revision, attributes)
+
+      render partial: self.class.top_level_description,
+             locals: { edition: edition, attachment: attachment }
+    end
+
+    it "renders a command paper with its number" do
+      render_with_attachment(official_document_type: "command_paper", paper_number: "123")
+      expect(rendered).to have_content("Ref: 123")
```

Comment:
This isn't a valid command paper number as it's missing a prefix. Not a problem for the renderer but we should fix the example to be valid.

Reply:
I understand that it's desirable to use representative data in the tests sometimes. My preference is to use minimal test data, especially where there's nothing to enforce it remains 'valid'. One advantage is that 'obviously invalid/artificial' data like 'my-org-id' or '123' is never surprising, whereas superficially valid data may become less so over time. Another advantage is that it slightly suggests the lack of complexity in the code under test.

I'd like to leave this as-is. Would that cause an issue, do you think?